packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -98,6 +98,11 @@ const PatientInfo: React.FC<PatientInfoProps> = ({ patient }) => {
         <span className={styles.patientName}>{name} </span>
       )}
       <span className={styles.patientInfo}>{`${age(patient?.birthDate)}, ${getGender(patient?.gender)}`}</span>
+      <ExtensionSlot
+        name="visit-header-tags-slot"
+        state={{ patientUuid: patient?.id, patient }}
+        className={styles.tagsFlexRow}
+      />
```

Comment:
Is that class necessary? It seems like the right styles are being brought by the visit-attribute-tags component.

I would rather this slot be called `visit-header-middle-slot`. Better to be less opinionated about what goes in it. 

Maybe eventually we could put the stuff below, from queue entry, into the slot as extensions—the code below feels a bit out of place. But that's a different PR.